index.d.ts

```diff
@@ -7,6 +7,10 @@ declare namespace serializeError {
 		message?: string;
 		code?: string;
 	} & JsonObject;
+
+	type Options = {
+		maxDepth: number;
```

Comment:
Needs a doc comment with a code example. See how to do that here: https://github.com/sindresorhus/typescript-definition-style-guide#documentation